src/cmake-tools.ts

```diff
@@ -746,7 +823,10 @@ export class CMakeTools implements vscode.Disposable, api.CMakeToolsAPI {
   private async _handleCacheEditorMessage(method: string, params: {[key: string]: any}): Promise<any> {
     switch (method) {
     case 'getEntries': {
-      const drv = await this._cmakeDriver;
+      const drv = await this.getCMakeDriverInstance();
+      if (!drv) {
+        return null;
```

Comment:
I think this should be returned as a `Promise`:
```ts
return Promise.resolve(null);
```

Reply:
Style guide question